tokio-net/src/tracing.rs

```diff
@@ -25,7 +25,7 @@ macro_rules! trace {
 
 #[cfg(not(feature = "tracing"))]
 macro_rules! trace {
-    ($($arg:tt)+) => {};
+    ($($arg:tt)+) => {{}};
```

Comment:
Why are these necessary?

Reply:
Otherwise, you get compile errors where `error!()` is used in an expression position. Using `()` would probably have worked as well... but `{}` seemed most appropriate.

Reply:
Oh, sure, I'm aware of the general case, just curious why you included this as a part of this change.

Reply:
I discovered the issue trying to figure out why doc gen didn't pass. It came up when I added a test for compiling `tokio-net` with only the `process` feature flag.